index.html

```diff
@@ -10,23 +10,26 @@
   <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
   <script src="https://kit.fontawesome.com/138cf4be1d.js" crossorigin="anonymous"></script>
+  <script src="myScript.js" async></script>
   <title>Lucas Comamala Portfolio</title>
 </head>
 
 <body>
 
 
-  <header>
-    <div class="topnav" id="myTopnav">
+  <header class="topnav" id="myTopnav">
       <a class="logo" href="#">Lucas Comamala</a>
       <div id="nav-links">
-        <a href="#works" class="active">Portfolio</a>
-        <a href="#about">About</a>
-        <a href="#contact">Contact</a>
+        <a class="nav-link" href="#works">Portfolio</a>
+        <hr class="menu-separator">
+        <a class="nav-link" href="#about">About</a>
+        <hr class="menu-separator">
+        <a class="nav-link" href="#contact">Contact</a>
+        <hr class="menu-separator">
       </div>
-      <a class="bars" href="#" aria-label="Webpage Navigation Links"><i class="fa fa-bars"></i></a>
-    </div>
+      <a id="bars" class="bars" href="#" aria-label="Webpage Navigation Links">&#9776;</a>
   </header>
```

Comment:
### Figma Template
![figma-menu (copy)](https://user-images.githubusercontent.com/93039359/179678793-561746c4-11e9-41ac-a799-d57a251db8a7.png)

- [ ] Your menu is working as expected :+1: But it would be nice if you add the bottom border and white overlay in your design :blush: